code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -414,15 +414,18 @@
 		overlay.appearance_flags = RESET_COLOR
 		add_overlay(overlay)
 	playsound(src, 'sound/effects/gravhit.ogg', 100, TRUE)
-	for (var/mob/living/candidate in view(gust_distance, src))
-		if(candidate == src || candidate.faction_check_mob(src))
+	for (var/turf/epicenter in view(1, usr.loc))
+		if(istype(epicenter, /turf/closed)) //Gusts dont go through walls.
 			continue
-		visible_message("<span class='boldwarning'>[candidate] is knocked back by the gust!</span>")
-		to_chat(candidate, "<span class='userdanger'>You're knocked back by the gust!</span>")
-		var/dir_to_target = get_dir(get_turf(src), get_turf(candidate))
-		var/throwtarget = get_edge_target_turf(target, dir_to_target)
-		candidate.safe_throw_at(throwtarget, 10, 1, src)
-		candidate.Paralyze(50)
+		for (var/mob/living/candidate in view(gust_distance, epicenter))
```

Comment:
I'd add candidate to a list and check to prevent them being gusted 5 times in a row